src/services.rs

```diff
@@ -134,13 +134,14 @@ impl ChannelKind {
     /// Get the type of values used to communicate with this service.
     pub fn get_type(&self) -> Type {
         use self::ChannelKind::*;
-        use values::Type::*;
+        use values::Type as ValueType;
```

Comment:
You mean just keep ```use values::Type;``` and reference members as ```Type::OnOff```? No specific reason, just wanted to stress that we are talking about value types not some generic types :) Both work for me actually